test/gie/builtins.gie

```diff
@@ -509,7 +509,7 @@ Central Conic
 ===============================================================================
 
 -------------------------------------------------------------------------------
-operation +proj=pipeline +step +proj=ccon +lat_1=52 +lat_0=52 +lon_0=19 +a=6390000 +x_0=330000 +y_0=-350000 +step +proj=axisswap +order=1,-2,3,4
+operation +proj=pipeline +a=6390000 +es=0 +step +proj=ccon +lat_1=52 +lat_0=52 +lon_0=19 +x_0=330000 +y_0=-350000 +step +proj=axisswap +order=1,-2
```

Comment:
Perhaps `+R=639000` is the correct thing to use here. `+es=0` shouldn't be necessary I think.

Reply:
Yes - this is leftovers from debugging. `R=6390000` is probably what should be used